src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/DefaultButton.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import Icon from '../../components/Icon';
+import type {DefaultButtonType} from './types';
+import React from 'react';
+import buttonStyles from './button.scss'; 
+import classNames from 'classnames';
+
+const ICON_ARROW_DOWN = 'chevron-down';
+
+export default class DefaultButton extends React.PureComponent {
+    props: DefaultButtonType;
+
+    static defaultProps = {
+        disabled: false,
+        hasOptions: false,
```

Comment:
`hasOptions` seems wrong to me, as the button itself never owns options. What do you think about `trailingIcon` or something like that.